posts/inside-rust/2020-10-15-Backlog-Bonanza.md

```diff
@@ -0,0 +1,107 @@
+author: Nicholas Matsakis
+team: the lang team <https://www.rust-lang.org/governance/teams/lang>
+---
+
+A month or two back, the lang team embarked on a new initiative that
+we call the "Backlog Bonanza". The idea is simple: we are holding a
+series of meetings in which we go through every pending RFC, one by
+one, and try to reach some sort of determination about what to do with
+it.  Once we've finished that, we can start in on categorizing other
+forms of backlog, such as tracking issues.
+
+### Possible outcomes for each RFC
+
+When we look at an RFC, we're typically deciding between one of the following outcomes:
+
+* **Close** the RFC, if we think that this problem is a good fit for the moment.
+* Close, but **suggest a [project proposal]**, if we think that the we might like to see the problem solved, but we aren't sure if the RFC has the design right, or we're not sure who would be a good liaison.
+* **Merge** the RFC, if we think the RFC basically nailed it and we have a lang team liaison in mind.
+
+[project proposal]: https://lang-team.rust-lang.org/proposing_a_project.html
+
+### Wait, what is a project proposal?
+
+I'm so glad you asked! The lang team is experimenting with a new
+process for extending the language. Instead of starting out by writing
+an RFC, the idea is to start with a **[project proposal]**. This is a
+lightweight description that you do by [opening an issue] on the
+[lang-team repository] using the "Project proposal" template. That
+will create an issue and a corresponding stream on Zulip.
+
+[lang-team repository]: https://github.com/rust-lang/lang-team/
+[opening an issue]: https://github.com/rust-lang/lang-team/issues/new/choose
+
+In our [weekly triage meetings], we go over each new project proposal
+and try to provide feedback. Project proposals generally result in one
+of a few possible outcomes:
+
+* **Close**, if we feel like the idea isn't a good fit right now.
+* **Suggest implementing**, if we feel like the idea is simple or obvious enough that an RFC isn't really needed. In that case, folks can just write a PR and we can use the fcp process to approve the PR.
+* **Charter a project group**, if we feel like the idea is good, but we'd like to see the design spelled out. To do this, there has to be some lang-team liaison who wants to help see it through (though that liaison doesn't have to be a member of the team; [serving as a liaison is a good way to get more involved in the lang team][path]).
+
+[weekly triage meetings]: https://lang-team.rust-lang.org/meetings.html
+[path]: https://blog.rust-lang.org/inside-rust/2020/07/09/lang-team-path-to-membership.html
+
+### Chartering a project group
+
+A "project group" is basically just a group of people working together
+completing some idea. Project groups are often pretty small, just 1 or 2
+people, though they can get significantly larger.
+
+Creating a smaller project group is meant to be lightweight. We
```

Comment:
I didn't really go into the idea that we have some project groups that are more formal and some that are less. I'm wondering about this distinction and how I feel about it. =)

I am wondering if I should avoid calling things "project groups" and just talk about "projects", where sometimes we also have a group associated with it and sometimes we don't.

Or maybe we should just make it easier to create a project group on the team repo and run with it.